app/controllers/users_controller.rb

```diff
@@ -0,0 +1,62 @@
+require './config/environment'
+
+class UsersController < ApplicationController
+
+#Signup Page
+#does not let a logged in user view the signup page
+  get '/signup' do
+    if logged_in?
+      redirect to '/tweets'
+    else
+      erb :'/users/create_user'
+    end
+  end
+
+#Signup Page- Form Submit
+  post '/signup' do
+    if params[:username] == "" || params[:email] == "" || params[:password] == ""
```

Comment:
I think you could use [ActiveRecord validations](http://guides.rubyonrails.org/active_record_validations.html) (e.g. `validates_presence_of :username`) to perform these types of checks more easily.